fem/lininteg.hpp

```diff
@@ -119,6 +119,33 @@ class DomainLFIntegrator : public DeltaLFIntegrator
    using LinearFormIntegrator::AssembleRHSElementVect;
 };
 
+
+class DomainLFGradIntegrator : public DeltaLFIntegrator
```

Comment:
Should probably have a comment like the others

Reply:
Comment added